jobs/circle_ci.rb

```diff
@@ -1,7 +1,8 @@
-require 'circleci'
+# frozen_string_literal: true
```

Comment:
redundant?

Reply:
It's needed (and encouraged by rubocop) - it's to make strings immutable.

https://stackoverflow.com/questions/37799296/ruby-what-does-the-comment-frozen-string-literal-true-do